subsys/logging/Kconfig.backends

```
@@ -3,20 +3,56 @@
 
 menu "Backends"
 
-config LOG_BACKEND_UART
+menuconfig LOG_BACKEND_UART
 	bool "Enable UART backend"
 	depends on UART_CONSOLE
 	default y if !SHELL_BACKEND_SERIAL
 	help
 	  When enabled backend is using UART to output logs.
 
+if LOG_BACKEND_UART
+
+config LOG_BACKEND_UART_OUTPUT_DICTIONARY
+	bool
+	depends on LOG2
+	select LOG_DICTIONARY_SUPPORT
+	help
+	  UART backend is in dictionary-based logging output mode.
+
+choice
+	prompt "UART Output Mode"
+	default LOG_BACKEND_UART_OUTPUT_TEXT
+
+config LOG_BACKEND_UART_OUTPUT_TEXT
+	bool "Text"
+	help
+	  Output in text.
+
 config LOG_BACKEND_UART_SYST_ENABLE
 	bool "Enable UART syst backend"
```

Thanks for making a choice out of this.
But when making a choice, I think it would be nice if this prompt was updated so the look-n-feel in whole choice group is aligned.

```suggestion
	bool "MIPI SyS-T"
```